Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/Visuals/ScaleHandles.cs

```diff
@@ -16,8 +16,17 @@ public class ScaleHandles : HandlesBase
         protected override HandlesBaseConfiguration BaseConfig => config;
         private ScaleHandlesConfiguration config;
         private bool areHandlesFlattened = false;
+
+        /// <summary>
+        /// Cached handle positions - we keep track of handle positions in this array
+        /// in case we have to reload the handles due to configuration changes.
+        /// </summary>
+        protected Vector3[] HandlePositions { get; private set; }
+        private int NumHandles => 8;
```

Comment:
does this need to be a property? or could it be a const field?

```suggestion
        private const int NumHandles = 8;
```

Reply:
@keveleigh I did this, sort of cargo-culting the way that the per-axis handles do it. The reason the per-axis handles do this is because they define the `NumHandles` as the length of the axis array:

```
private int NumHandles => handleAxes.Length;
```

But I don't see much of a reason for this to be done the same way in ScaleHandles. I'll do this change, but I need to make sure that it won't cause any weird side effects.

Reply:
No ill effects from this change. Committing.